app/controllers/workflows_controller.rb

```diff
@@ -57,7 +51,13 @@ def create
     redirect_to solr_document_path(cocina_object.externalIdentifier), notice: msg
   end
 
-  # add a workflow to an object if the workflow is not present in the active table
+  ##
+  # Updates the status of a specific workflow process step to a given status.
+  #
+  # @option params [String] `:item_id` The druid for the object.
+  # @option params [String] `:id` The workflow name. e.g., accessionWF.
+  # @option params [String] `:process` The workflow step. e.g., publish.
+  # @option params [String] `:status` The status to which we want to reset the workflow.
   def update
```

Comment:
all changes above are just moving the comments around since I think they were all in the wrong place for some reason